qualcoder/GUI/ui_visualise_graph.py

```diff
@@ -17,8 +17,6 @@ def setupUi(self, Dialog_visualiseGraph):
         self.splitter = QtWidgets.QSplitter(Dialog_visualiseGraph)
         self.splitter.setOrientation(QtCore.Qt.Vertical)
         self.splitter.setObjectName("splitter")
-        self.graphicsView = QtWidgets.QGraphicsView(self.splitter)
```

Comment:
Ok for these changes.  I will have to update the UI files too to match, otherwise these will get overridden in the future.

Reply:
i understand that, but i'm not familar with the qt designer, i tried it but did not manage to achive what i wanted.